web-console/Storage/OCS/OCS _Installation_Workflow.md

```diff
@@ -0,0 +1,78 @@
+# OCS Install Workflow
+
+### OperatorHub
+
+There will be 2 OCS operator entries in OperatorHub covering each of the following modes. 
+* Converged Mode -- Local Cluster / Hyperconverged 
+* Independent Mode -- External Cluster (Ceph Cluster is deployed externally)
+
+### Converged Mode Design
+
+OperatorHub
+![OperatorHub operators view](img/OCS-operator-view.png)
+
+Installation
+![OperatorHub operators view](img/OCS-Install-step1.png)
+
+![OperatorHub operators view](img/OCS-Install-step2.png)
+
+In the installed operator view the user can see the OCS operator installed. 
+![OperatorHub operators view](img/Installed_operators_list.png)
+
+![subscription operators page](img/OCS_Subscription_page.png) 
+
+# The OCS installation process
+OCS Overview page
+![OCS-View](img/OCS-View.png)
+
+Clicking on "create new" (OCS Service) Will start the following installation flow: 
+
+Same flow will be available by clicking on the OCS tab and "Create new" button. 
+
+### Select Nodes
+Admin needs to select 3 nodes (depends on the requested capacity input) to label with “cluster.ocs.openshift.io/openshift-storage=""” (note label is subject to change as this will be automatically configured by the OCS operator) to be used for the OCS cluster.
```

Comment:
at least 3 nodes. Remove 'requested capacity input' - there is no such input)

The label is performed by the OCS installer, not the operator.

Reply:
There is a requested capacity input, the user can request usable capability in 1 TiB increments. Those are the requirements.

Reply:
Nope. We (I) removed it. We give 1TiB initially, no questions asked.